tests/testthat/test-subscript-tables.R

```diff
@@ -226,6 +226,190 @@ test_that("drop and exact recognised and used appropriately", {
     expect_error(x.6.5.named[[2, 3, exact = "TRUE"]], "exact argument should be TRUE or FALSE")
 })
 
+tbls <- readRDS("qTablesWithZStatInCells.rds")
+
```

Comment:
The current tests ignore all the other data.frame columns involving the significance (arrows size, multiplier, direction etc). I feel the tests could be improved by checking if the attributes have those columns and then inspecting that the z-statistic has the correct value?

Reply:
See 6f9cc2fd54